mleap-spring-boot/src/main/scala/ml/combust/mleap/springboot/ModelLoader.scala

```diff
@@ -0,0 +1,52 @@
+package ml.combust.mleap.springboot
+
+import TypeConverters._
+import javax.annotation.PostConstruct
+import org.slf4j.LoggerFactory
+import ml.combust.mleap.pb
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+import scala.collection.JavaConverters._
+import java.nio.file.{Paths, Files, Path}
+import scalapb.json4s.Parser
+
+@Component
+class ModelLoader {
```

Comment:
We should autowire the mleap executor and parser that get created in the StarterConfiguration and use them in this class 

```
class ModelLoader(@Autowired val mleapExecutor: MleapExecutor,
                  @Autowired val jsonParser: Parser)
```

instead of creating a new parser
```
private val jsonParser = new Parser()
```

and doing

```
StarterConfiguration.getMleapExecutor
```

I believe the getMleapExecutor() is then not needed?